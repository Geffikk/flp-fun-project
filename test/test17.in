S,A
a,b
S
S->AbA
S->bA
S->Ab
S->b
A->Aa
A->a
