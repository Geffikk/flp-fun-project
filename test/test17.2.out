S,A,<bA>,b',a'
a,b
S
S->A<bA>
<bA>->b'A
b'->b
S->b'A
S->Ab'
S->b
A->Aa'
a'->a
A->a
